cmake/projects/tcl/schemes/url_sha1_tcl_autotools.cmake.in

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include("@HUNTER_SELF@/cmake/Hunter")
+
+include(hunter_user_error)
+include(hunter_status_debug)
+include(hunter_assert_not_empty_string)
+include("@HUNTER_PACKAGE_SETUP_DIR@/schemes/hunter_autotools_project.cmake")
+
+hunter_status_debug("Scheme: url_sha1_autotools")
+
+if("@MSVC@")
+  hunter_user_error("Autotools scheme not supported with Visual Studio")
+endif()
+
+foreach(dependency ${DEPENDS_ON_PACKAGES})
+  hunter_add_package(${dependency})
+endforeach()
+
+# Check preconditions
+hunter_assert_not_empty_string("@HUNTER_SELF@")
+hunter_assert_not_empty_string("@HUNTER_EP_NAME@")
+hunter_assert_not_empty_string("@HUNTER_PACKAGE_URL@")
+hunter_assert_not_empty_string("@HUNTER_PACKAGE_SHA1@")
+hunter_assert_not_empty_string("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_assert_not_empty_string("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_assert_not_empty_string("@HUNTER_PACKAGE_BUILD_DIR@")
+hunter_assert_not_empty_string("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+hunter_assert_not_empty_string("@HUNTER_INSTALL_PREFIX@")
+
+set(cflags)
+set(extra_flags)
+set(patch_command
+  ${CMAKE_COMMAND} -E copy "@HUNTER_PACKAGE_SCRIPT_DIR@/configure" "@HUNTER_PACKAGE_SOURCE_DIR@"
+)
+
+if(ANDROID)
+  set(KERNEL_VERSION 3) # API 14-23
+  if (${CMAKE_SYSTEM_VERSION} GREATER_EQUAL "24")
+    set(KERNEL_VERSION 4) # API 24-x
+  endif()
+  configure_file("@HUNTER_PACKAGE_SCRIPT_DIR@/android_uname.in" "@HUNTER_PACKAGE_DOWNLOAD_DIR@/uname" @ONLY)
+
+  list(APPEND cflags "--include android_def.h")
+  list(APPEND extra_flags "--enable-langinfo=no")
+  list(APPEND patch_command
+    && ${CMAKE_COMMAND} -E copy "@HUNTER_PACKAGE_SCRIPT_DIR@/android_def.h" "@HUNTER_PACKAGE_SOURCE_DIR@"
```

Comment:
Please try not to use `&&`:
* https://github.com/ruslo/hunter/issues/246

You can extend ExternalProject_Add step by `COMMAND`, I think it should work here.

Reply:
substituted && by COMMAND
build ok https://travis-ci.com/drodin/hunter/builds/81224211